lib/js/components/ProgressBar/ProgressBar.vue

```diff
@@ -44,6 +44,7 @@
 				:style="{ left: range.percentValueFrom + '%', width: range.length + '%' }"
 			>
 			</div>
+			<div class="progressBar__resultsShadow"></div>
```

Comment:
you know, I don't non-meaningful, not required HTML markup. Can we get rid of it?

Reply:
> you know, I don't non-meaningful, not required HTML markup. Can we get rid of it?
It's not clear for me. How You will add shadow to whole progressbar? On parent it doesn't work (layers are above), when I add to single layer, we have error https://bethinkteam.slack.com/archives/CFBLF5RGA/p1664803900641519?thread_ts=1664532308.131999&cid=CFBLF5RGA , this markup is a fix for this.

Reply:
did you try `::after`?

Reply:
@kvas-damian of course it's possible, I was thinking about it and i chose HTML tag. I do not insist on him, but I also do not see any advantage in doing it by :after. What's your argument, shorter/clear HTML? Shorter HTML but more css rules.

Reply:
every HTML element means additional additional DOM node in DOM tree. When the tree is big, traversing and updating it is more computing heavy. Pseudoelements aren't put in DOM, so they don't affect that area.

Additional argument is separation of code layers. When you use `::after` then styles belonging to an element are kept in one place, when you put them in additional HTML element, then the logic is distributed and have to be maintained in two places.

Reply:
Changed.